tests/bgp/test_4-byte_asn_community.py

```diff
@@ -0,0 +1,243 @@
+    dut_ipv6_network = duthost.shell("show run bgp | grep 'ipv6 prefix-list'")['stdout'].split()[6]
+    neigh_ipv4_network = nbrhosts[neigh]["host"].shell("show run bgp | grep 'ip prefix-list'")['stdout'].split()[6]
+    neigh_ipv6_network = nbrhosts[neigh]["host"].shell("show run bgp | grep 'ipv6 prefix-list'")['stdout'].split()[6]
+
+    setup_info = {
+        'duthost': duthost,
+        'neighhost': neighbors[neigh],
+        'neigh': neigh,
+        'dut_asn': dut_asn,
+        'neigh_asn': neigh_asn[neigh],
+        'asn_dict':  neigh_asn,
+        'neighbors': neighbors,
+        'cli_options': cli_options,
+        'neigh_cli_options': neigh_cli_options,
+        'dut_ip_v4': dut_ip_v4,
+        'dut_ip_v6': dut_ip_v6,
+        'neigh_ip_v4': neigh_ip_v4,
+        'neigh_ip_v6': neigh_ip_v6,
+        'peer_group_v4': peer_group_v4,
+        'peer_group_v6': peer_group_v6,
+        'asic_index': asic_index,
+        'dut_bgp_id': dut_bgp_id,
+        'neigh_bgp_id': neigh_bgp_id,
+        'dut_ipv4_network': dut_ipv4_network,
+        'dut_ipv6_network': dut_ipv6_network,
+        'neigh_ipv4_network': neigh_ipv4_network,
+        'neigh_ipv6_network': neigh_ipv6_network
+    }
+
+    logger.info("DUT BGP Config: {}".format(duthost.shell("show run bgp", module_ignore_errors=True)['stdout']))
+    logger.info("Neighbor BGP Config: {}".format(nbrhosts[neigh]["host"].shell("show run bgp")['stdout']))
+    logger.info('Setup_info: {}'.format(setup_info))
+
+    yield setup_info
+
+    # restore config to original state
+    config_reload(duthost)
+    config_reload(neighbors[neigh], is_dut=False)
+
+    # verify sessions are established
+    bgp_facts = duthost.bgp_facts(instance_id=asic_index)['ansible_facts']
+    for k, v in bgp_facts['bgp_neighbors'].items():
+        if v['description'].lower() not in skip_hosts:
+            logger.debug(v['description'])
+            assert v['state'] == 'established'
+
+
+def test_4_byte_asn_community(setup):
+    cmd = 'vtysh {} \
+    -c "config" \
+    -c "no router bgp {}" \
+    -c "router bgp {}" \
+    -c "bgp router-id {}" \
+    -c "bgp log-neighbor-changes" \
+    -c "no bgp ebgp-requires-policy" \
+    -c "no bgp default ipv4-unicast" \
+    -c "bgp bestpath as-path multipath-relax" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} peer-group" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "neighbor {} remote-as {}" \
+    -c "neighbor {} peer-group {}" \
+    -c "neighbor {} description {}" \
+    -c "neighbor {} timers 3 10" \
+    -c "neighbor {} timers connect 10" \
+    -c "address-family ipv4 unicast" \
+    -c "network {}" \
+    -c "neighbor {} soft-reconfiguration inbound" \
+    -c "neighbor {} route-map FROM_BGP_PEER_V4 in" \
```

Comment:
route-map define is missing in the file, what exactly the route-map contain, similar other route map in the files as well.

Reply:
It is an existing route-map.  I added a comment specifying this.